app/test.js

```diff
@@ -0,0 +1,44 @@
+const { getColorFromPairNumber, getPairNumberFromColor, ColorPair, colorManual } = require( "./color_number");
+
+function test(){
+	    pairNumber = 4;
+            let testPair1 = getColorFromPairNumber(pairNumber);
+            console.log(`[In]Pair Number: ${pairNumber},[Out] Colors:${testPair1}`);
+			console.assert(testPair1.majorColor == "WHITE");
+			console.assert(testPair1.minorColor == "BROWN");
+
+			pairNumber = 5;
+            testPair1 = getColorFromPairNumber(pairNumber);
+            console.log(`[In]Pair Number: ${pairNumber},[Out] Colors:${testPair1}`);
+			console.assert(testPair1.majorColor == "WHITE");
+			console.assert(testPair1.minorColor == "SLATEGRAY");
+			 
+			pairNumber = 23;
+            testPair1 = getColorFromPairNumber(pairNumber);
+            console.log(`[In]Pair Number: ${pairNumber},[Out] Colors:${testPair1}`);
+			console.assert(testPair1.majorColor == "RED");
+			console.assert(testPair1.minorColor == "GREEN");
+			 
+            let testPair2 = new ColorPair();
+			testPair2.majorColor="YELLOW";
+			testPair2. minorColor ="GREEN";
+            pairNumber =getPairNumberFromColor(testPair2);
+            console.log(`[In]Colors: ${testPair2}, [Out] PairNumber: ${pairNumber}`);
+			console.assert(pairNumber==18);
+
+			testPair2 = new ColorPair();
+			testPair2.majorColor="RED";
+			testPair2. minorColor ="BLUE";
+            pairNumber =getPairNumberFromColor(testPair2);
+            console.log(`[In]Colors: ${testPair2}, [Out] PairNumber: ${pairNumber}`);
+			console.assert(pairNumber==6);
+}
+
+function printManual() {
+    let manual = colorManual();
+    console.log('\nColor Coding Reference Manual:\n');
+    console.log('PairNumber           Colors\n');
```

Comment:
The file is about testing functionality. But this line prints the table heading, which is production code.
That means a customer who wants reference-manual functionality will need to take the entire test code in this file.
It's like sending ISRO's test-equipment along with Chandrayan to the moon :)
Always separate test code from production code